### PR TITLE
Add TerraMind model

### PR DESCRIPTION
**Pull request by yawenzzzz**
https://huggingface.co/ibm-esa-geospatial/TerraMind-1.0-base

### CODE REVIEW

**Review by favyen2 — APPROVED** [review]
I pushed fix for self.model_size and make norm support stacked time series

File: rslearn/models/terremind.py [hunk 1]
```diff
@@ -0,0 +1,238 @@
+"""Terremind models."""
```
[the remark this answers is not in the document]
Comment (yawenzzzz) [reply rslearn/models/terremind.py:1]: Fixed.

File: rslearn/models/terremind.py [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+}
+
+
+class Terramind(torch.nn.Module):
+    """Terremind backbones."""
+
+    def __init__(
+        self,
+        model_size: TerramindSize,
+        image_size: int = DEFAULT_IMAGE_SIZE,
```
[the remark this answers is not in the document]
Comment (yawenzzzz) [reply rslearn/models/terremind.py]: Fixed.

File: rslearn/models/terremind.py [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+        2987.011,
+        3083.234,
+        3132.220,
+        3162.988,
+        2424.884,
+        1857.648,
+    ],
+    "S1GRD": [-12.599, -20.293],
+    "S1RTC": [-10.93, -17.329],
+    "RGB": [87.271, 80.931, 66.667],
```
[the remark this answers is not in the document]
Comment (yawenzzzz) [reply rslearn/models/terremind.py]: OK, i added information for each modality.

File: rslearn/models/terremind.py [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+            depth = 1024
+        else:
+            raise ValueError(f"Invalid model size: {self.model_size}")
+        return [(PATCH_SIZE, depth)]
+
+
+class TerramindNormalize(Transform):
+    """Normalize inputs using Terramind normalization.
+
+    It will apply normalization to the modalities that are specified in the model configuration.
```
[the remark this answers is not in the document]
Comment (yawenzzzz) [reply rslearn/models/terremind.py]: Fixed, information added for S1, for both GRD and RTC, the unit if dB, so we should convert to decibel to be compatible with their normalization values.
